marketpulse/main/migrations/0013_auto_20150507_1008.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from slugify import slugify
+
+from django.db import models, migrations
+
+from marketpulse.main import FFXOS_MEDIA_ACTIVITY_NAME
+
+
+def new_activity(apps, schema_editor):
+    Activity = apps.get_model("main", "Activity")
+    Activity.objects.create(name=FFXOS_MEDIA_ACTIVITY_NAME,
+                            slug=slugify(FFXOS_MEDIA_ACTIVITY_NAME))
```

Comment:
You don't need the `slug` here. The save method in models.py will handle the slug creation.